addons/calendar/models/res_users.py

```diff
@@ -13,22 +14,26 @@ def _systray_get_calendar_event_domain(self):
         start_dt = datetime.datetime.utcnow()
         start_date = datetime.date.today()
```

Comment:
start_date should be in the user timezone, otherwise it might generate a wrong date